stylus/settings/palette.json

```diff
@@ -44,5 +16,33 @@
   "info": "var(--infoColor)",
   "errorBackground": "var(--errorBackground)",
   "primaryBackgroundLight": "var(--primaryBackgroundLight)",
-  "neutralBackground": "var(--neutralBackground)"
+  "neutralBackground": "var(--neutralBackground)",
+  "zircon": "#F5FAFF",
+  "frenchPass": "#C2DCFF",
+  "dodgerBlue": "#297EF2",
+  "scienceBlue": "#0B61D6",
+  "puertoRico": "#0DCBCF",
+  "grannyApple": "#DEF7E7",
+  "emerald": "#35CE68",
+  "malachite": "#08b442",
+  "seafoamGreen": "#3DA67E",
+  "brightSun": "#FFC644",
+  "texasRose": "#FFAE5F",
+  "chablis": "#FFF2F2",
+  "yourPink": "#FDCBCB",
+  "fuchsia": "#FC4C83",
+  "pomegranate": "#F52D2D",
+  "monza": "#DD0505",
+  "portage": "#9169F2",
+  "azure": "#1FA8F1",
+  "melon": "#FD7461",
+  "blazeOrange": "#FC6D00",
+  "mango": "#FF962F",
+  "pumpkinOrange": "#FF7F1B",
+  "lavender": "#C2ADF4",
+  "darkPeriwinkle": "#6984CE",
+  "purpley": "#7F6BEE",
+  "lightishPurple": "#B449E7",
+  "barney": "#922BC2",
+  "weirdGreen": "#40DE8E"
 }
```

Comment:
Because Avatars use them, but we do not want to have the CSS vars as part of our CSS properties because then people start to use them too easily.

Reply:
> Also, the CSS custom properties are there whether you use the colors or not

For my personal culture, is it bad? Can we have perf issue with lot of vars? 

> we do not want to have the CSS vars as part of our CSS properties because then people start to use them too easily

 We want to make it "harder" for the dev to use a custom palette colors to force him/her to think about semantic colors first. Clear enough, thanks!

Reply:
:+1: Sorry if it was unclear 😅 !

Reply:
Also I think it could be confusing to have CSS custom properties that are meant to be overrided and others that should not be.